yugabyte/src/yugabyte/append.clj

```diff
@@ -9,11 +9,29 @@
             [jepsen.tests.cycle :as cycle]
             [jepsen.tests.cycle.append :as append]))
 
+(defn workload-rr
```

Comment:
This is a snapshot, NOT repeatable read.
Differences aren't big but they should matter for Jepsen.
Is there a snapshot consistency model?

Reply:
I also though about that. Well this is kind of user-faced configuration, if we define RR we expect RR. Expecting other consistency model is not clear.
But can change, sure.

Reply:
Fixed